src/main/java/de/hpi/bpt/argos/persistence/database/PersistenceEntityManagerImpl.java

```diff
@@ -268,23 +268,41 @@ public Event createEvent(long eventTypeId, String requestBody) {
 			return null;
 		}
 
-		ProductConfiguration configuration = getProductConfiguration(product,
-				getCodingPlugIdentifier(event.getEventData()),
-				getCodingPlugSoftwareVersion(event.getEventData()));
+		long productConfigurationId;
+		ProductConfiguration configuration;
+
+		try {
+		    productConfigurationId = product.getProductConfiguration(
+                    getCodingPlugIdentifier(event.getEventData()), getCodingPlugSoftwareVersion(event.getEventData()))
+                    .getId();
+            configuration = databaseConnection.getProductConfiguration(productConfigurationId);
+        } catch (NullPointerException e) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1166)